var/spack/repos/builtin/packages/r-matrix/package.py

```diff
@@ -32,7 +32,9 @@ class RMatrix(RPackage):
     homepage = "http://matrix.r-forge.r-project.org/"
     url      = "https://cran.rstudio.com/src/contrib/Matrix_1.2-11.tar.gz"
     list_url = "https://cran.r-project.org/src/contrib/Archive/Matrix"
-    version('1.2-11', 'a8c1a893f36d7ea918ddbf8cb8d10b43')
+
+    version('1.2-12', '0ade6e374716f08650cc8b8da99a313c')
+    version('1.2-11', 'b7d2a639aa52228dfde7c3c3ee68b38e')
```

Comment:
Sidenote: any idea why the checksum changed for the same version?

Reply:
I don't know. But with 'spack checksum' the checksum for 1.2-11 changed.